cli/lib/kontena/cli/common.rb

```diff
@@ -11,11 +11,11 @@ module Cli
     module Common
 
       def logger
-        return @logger if @logger
-        @logger = Logger.new(ENV["DEBUG"] ? $stderr : $stdout)
-        @logger.level = ENV["DEBUG"].nil? ? Logger::INFO : Logger::DEBUG
-        @logger.progname = 'COMMON'
-        @logger
+        Kontena.logger
+      end
+
+      def debug(msg_or_progname = nil, &block)
+        block_given? ? Kontena.logger.debug(msg_or_progname || Kontena.logger.progname, &block) : Kontena.logger.debug(msg_or_progname)
```

Comment:
See how the [kontena-ipam `Logging`](https://github.com/kontena/kontena-ipam/blob/master/app/logging.rb#L33-L35) class wraps these:

```ruby
  def debug(message = nil, &block)
    logger.add(Logger::DEBUG, message, logging_prefix, &block)
  end
```

OTOH why not have everything just use `logger.debug { ... }` instead?